vitest.config.ts

```
@@ -0,0 +1,20 @@
+    coverage: {
+      provider: 'v8',
+      reporter: ['text', 'json', 'html'],
+      exclude: [
+        'node_modules/',
+        'dist/',
+        '__tests__/',
+        '*.config.*',
+      ],
+    },
+    include: ['**/?(*.)+(spec|test).+(ts|tsx|js)'],
+    exclude: ['__tests__/client.test.ts', 'dist/', 'node_modules/'],
```

ESLint

[error] Strings must use doublequote.

(@typescript-eslint/quotes)